stable/fairwinds-insights/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "8.9.0"
 description: A Helm chart to run the Fairwinds Insights server
 name: fairwinds-insights
-version: 0.5.21
+version: 0.5.22
```

Comment:
actually can you make this `0.6.0` i think the support for insightsCI version `2.0` is not released yet.

Reply:
@makoscafee I'm going to be releasing it together on Wednesday's release. Is that still a problem?

Reply:
if they are released together then that is fine.